docs/src/cli/manage-stake-accounts.md

```diff
@@ -0,0 +1,74 @@
+# Manage Stake Accounts
+
+If you want to delegate stake to many different validators, you will need
+to create a separate stake account for each. If you follow the convention
+of creating the first stake account at seed "0", the second at "1", the
+third at "2", and so on, then the `solana-stake-accounts` tool will allow
+you to operate on all accounts with single innovations. You can use it to
```

Comment:
This is a strange usage of "innovations". Do you mean "invocations"?

Reply:
Yes, fixed